pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -2402,6 +2402,13 @@ struct pjsua_msg_data
      */
     pj_str_t    target_uri;
 
+    /**
+     * Optional local URI (i.e. From header). If NULL, the
+     * contact is used for the From header. This field is currently
```

Comment:
instead of `the contact`, shouldn't it be `the account ID`?

Reply:
Good point: the account ID defines the contact header and the from header. But the from header is not defined by the contact header.